### ISSUE TITLE
Screen Reader No Longer Reads Characters Such As Left-Brace And Right-Brace

### ISSUE THREAD

**Issue by munawarb** [microsoft/vscode#204151]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.86.0 (user setup)
- OS Version: Windows 11 Home

Steps to Reproduce:

1. Load a screen reader such as JAWS or NVDA.
2. Open a file (such as a Typescript file) that uses C-style braces `{}`
3. Observe that when reading lines containing braces, the screen reader doesn't announce them. For example, if a line reads `import { lib } from "test";`, the screen reader announces `import lib from quote test quote semicolon` when it should be saying `import left brace lib right brace from quote test quote semicolon`. This is with high punctuation verbosity turned on in the screen reader, so it should be announcing these characters as it does in other applications such as Notepad. In other words, the braces are not read out.
4. Further, when using left and right arrow to read character by character, when arrowing over a brace, the screen reader announces "blank" instead of "left brace" or "right brace."

**Comment by meganrogge** [comment]
/duplicate #200956
will be fixed in february

**Comment by munawarb** [comment]
I was able to fix it for now until the Feb update by downgrading to November 2023 recovery 2, for whoever comes here. Just be sure to disable automatic updates or the IDE will update as soon as you open the older version.
